lib/trento/application/usecases/clusters/clusters.ex

```diff
@@ -126,9 +126,10 @@ defmodule Trento.Clusters do
        }) do
     hosts_data =
       Repo.all(
-        from h in HostReadModel,
+        from(h in HostReadModel,
```

Comment:
why this change? we have a lot of entries without parenthesis

Reply:
Idk, I think the rebase did some magic, here

Reply:
please remove the parenthesis, the formatter did something wrong

Reply:
done